Cue/app/tabs/library/DeckView.android.js

```diff
@@ -98,28 +98,39 @@ class DeckView extends React.Component {
       onPress: () => { this.props.navigator.pop() }
     }
 
-    const rightItems = [
+    let rightItems = [
       {
         title: 'Filter flagged cards',
         icon: this.state.isFiltering ? CueIcons.filterToggleSelected : CueIcons.filterToggle,
         onPress: () => this.setState({
           isFiltering: !this.state.isFiltering
         })
-      },
-      {
-        title: 'Add card',
-        icon: CueIcons.plus
-      },
-      {
-        title: 'Deck sharing options',
-        icon: CueIcons.share,
-      },
-      {
-        title: 'Edit',
-        icon: CueIcons.edit
-      },
+      }
     ]
 
+    if (this.props.deck.accession === 'private') {
+      rightItems = rightItems.concat([
+        {
+          title: 'Add card',
+          icon: CueIcons.plus
+        },
+        {
+          title: 'Deck sharing options',
```

Comment:
Mmm, I mean that the Deck Sharing Options screen can still be opened, but it would be display only and can't be edited. So if the deck is Shared, it would only display the sharing code, and if the deck is Public, then it would only tell you that the deck is public.